### ISSUE TITLE
Preview not shown upon opening file / Entry editor keybinding flawed

### ISSUE THREAD

**Issue by matthiasgeiger** [JabRef/jabref#4808]
JabRef version 5.0dev

As reported by @AEgit in https://github.com/JabRef/jabref/issues/4762#issuecomment-474752885: The behavior of the preview and the entry editor keybinding is somehow broken.

- [x] I have tested the latest development version from http://builds.jabref.org/master/ and the problem persists

1. Having "View" -> "Entry Preview" checked and opening JabRef / a new database: No preview shown. Unchecking and checking again opens preview.

2. Opening Entry editor by keybinding CTRL - E is only working when preview or entry editor has been opened before

**Comment by tobiasdiez** [comment]
Refs #2115.

**Comment by harinda05** [comment]
Hi, I have started working on this issue. Will update the progress shortly

**Comment by harinda05** [comment]
I have fixed the first issue regarding the "Entry Preview" is not working correctly. I think above mentioned are two are two different issues. Does the PR has to be included with both the solutions? @tobiasdiez  any advice would be appreciated :)

**Comment by bernhard-kleine** [comment]
JabRef 5.0-dev--snapshot--2019-03-26--master--83c802269
Windows 7 6.1 amd64 
Java 1.8.0_201

I tested the ctrl-E, doubleclick on an entry and entry preview in todays version. Doubleclicking worked. However, trying ctrl-E to toggle the entry editor did not work, even when the entry editor was opened. That means Doubleclick and ctrl-e are only optically doing the same. After I clicked View->Open entry editor, Doubleclicking and Crtl could both toggle the entry editor.

The preview styles was blocked as long as I had not clicked View->Open entry editor. After clicking it alt-2 worked flawlessly.

My 5Cts

**Comment by tobiasdiez** [comment]
@harinda05 No, it is fine if your PR just fixes this bug here. I just guessed that fixing #2115 would also solve this bug (but of course #2115 is more complicated and requires more work).

**Comment by ElizBaldwin** [comment]
I am not sure if this is an additional comment to this bug, or if I should start a new one.  For me, the preview is always blank.  The section where it should be is present, but empty.

I am using Jabref 4.3.1 on Fedora 30.  I tried to see if the problem persists in the latest development version, but that will not run for me (should I submit a separate bug report on that?).  

My java -version is
openjdk version "1.8.0_212"
OpenJDK Runtime Environment (build 1.8.0_212-b04)
OpenJDK 64-Bit Server VM (build 25.212-b04, mixed mode)

this works in general with 4.3.1 and is supported according to the installation notes.

**Comment by Nicolarge** [comment]
@ElizBaldwin 
I've had the exact same problem until now using Jabref 4.3.1 on Fedora 30 with 1.8.0.222.b10-0.
On a side note, the same version of Jabref was working perfectly fine on Fedora 28.
I've realized that the libjfxwebkit.so was actually missing in the jre folder.
Luckily I had this library in my Matlab folder so I just copied it into: 
/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.222.b10-0.fc30.x86_64/jre/lib/amd64/libjfxwebkit.so
Problem solved. The entry preview works just as expected now (not blank anymore).

**Comment by ElizBaldwin** [comment]
@Nicolarge 
Thank you!  That fixed it!  I seemed to have the file in my jre installation but I think the Jabref notes say we have to use openjdk?  Anyway I copied it across and now I have the preview back.

So this is a weird bug in openjdk, that it's just missing a file?  I suppose we ought to report that?

**Comment by stefan-kolb** [comment]
Thank you for your report :+1:
This should be fixed in current `master`. Please try the latest build from http://builds.jabref.org/master.

**Comment by AEgit** [comment]
JabRef 5.0-alpha--snapshot--2019-08-25--master--0c342bdc7
Windows 10 10.0 amd64 
Java 1.8.0_221

I cannot confirm that the issue has been fully fixed. <kbd>CTRL</kbd> + <kbd>E</kbd> does not open the entry editor (it did the first time, but I have now opened the database again and it does not work anymore). To open the entry editor reliably, one has to doubleclick the respective entry. <kbd>CTRL</kbd> + <kbd>E</kbd> DOES close the entry editor, thought, when it it already open.

**Comment by AEgit** [comment]
JabRef 5.3--2021-01-04--10180ed
Windows 10 10.0 amd64 
Java 14.0.2

The issue mentioned in https://github.com/JabRef/jabref/issues/4808#issuecomment-524628653 persists in the current development version of JabRef.
